focus_ios/focus_ios.model.lkml

```diff
@@ -2,6 +2,15 @@ connection: "telemetry"
 label: "Firefox Focus for iOS"
 include: "//looker-hub/focus_ios/explores/*"
 # include: "//looker-hub/focus_ios/dashboards/*"
-# include: "views/*"
+include: "views/*"
 # include: "explores/*"
 # include: "dashboards/*"
+
+explore: usage {
+  always_filter: {
+    filters: [
+      submission_date: "26 weeks",
+    ]
+  }
+  sql_always_where: submission_date >= "2020-01-01" ;;
```

Comment:
This filter actually overrides the `AND submission_date >= DATE(2019,1,1)` part in the `derived_table` definition.